.github/workflows/build_test_invoke.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Run build tests for tests/unit/
         run: |
           pip install -r requirements.txt
-          pytest -vvv tests/unit/
+          pytest -n $(python -c "import psutil; print(psutil.cpu_count(logical=True))") -vvv tests/unit/
```

Comment:
Actually, we don't need `psutil` for this, we can just use `-n auto` flag from xdist directly. See https://pytest-xdist.readthedocs.io/en/stable/distribution.html

Reply:
thanks @mndeveci for you suggestion, let me update my PR

Reply:
updated